css/properties.json

```diff
@@ -4712,6 +4640,49 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset": {
+    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": [
+      "offset-distance"
+    ],
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": [
+      "offset-path",
+      "offset-distance",
+      "offset-rotation",
+      "offset-anchor"
+    ],
+    "appliesto": "allElementsSVGContainerElements",
+    "computed": [
+      "offset-path",
+      "offset-distance",
+      "offset-rotation",
+      "offset-anchor"
+    ],
+    "order": "orderOfAppearance",
```

Comment:
[The spec. says "per grammar"](https://drafts.fxtf.org/motion-1/#propdef-offset), so you need to replace this by [`"perGrammar"`](https://github.com/mdn/data/blob/fb3f805df8e167d8609aedadb72c30aba7b88c51/l10n/css.json#L1360).

Sebastian